manifests/init.pp

```diff
@@ -18,17 +18,12 @@
   String                  $bolt_metrics_ensure         = 'present',
   Array[String]           $bolt_hosts                  = puppet_metrics_collector::hosts_with_pe_profile('bolt_server'),
   Integer                 $bolt_port                   = 62658,
-  # Collection of ActiveMQ metrics has been removed, but the parameters are left to avoid breaking upgrades
```

Comment:
Are these supposed to be removed in this PR?

Reply:
Nope, good catch.  I forgot that I moved that work to a different PR and decided not to remove them.  I'll reconcile this.

Reply:
Since the AMQ PR is already merged, I just copied that patch over here.  Should be good now.